### PR TITLE
Ko/hc 329 js sdk deploy part 2

### PR DESCRIPTION
**Pull request by ibihim**

### CODE REVIEW

**Comment by s-urbaniak** [comment]
I ... uhm ....

**Comment by s-urbaniak** [comment]
https://media0.giphy.com/media/Pqr5ddlDQqI8g/giphy.gif

File: scripts/release_sdk.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Releases given version to github release indirectly by setting a tag and triggering travis 
+#
+####################################################################################################
+
+VERSION=$1
+
+git add . && \
+    git commit -m "releasing $VERSION" && \
+    git tag "$VERSION" -m "$(cat CHANGELOG.md | scripts/read_changelog.sh $VERSION)" && \
+    git push origin master --tag
```
Comment (s-urbaniak) [inline scripts/release_sdk.sh]: does this environment not have md5sum?

File: scripts/release_sdk.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+#
+# Releases given version to github release indirectly by setting a tag and triggering travis 
+#
+####################################################################################################
+
+VERSION=$1
+
+git add . && \
+    git commit -m "releasing $VERSION" && \
+    git tag "$VERSION" -m "$(cat CHANGELOG.md | scripts/read_changelog.sh $VERSION)" && \
```
Comment (s-urbaniak) [inline scripts/release_sdk.sh]: `${VERSION}`

File: scripts/release_sdk.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (s-urbaniak) [inline scripts/release_sdk.sh]: `#!/usr/bin/env bash`

**Comment by s-urbaniak** [comment]
please squash the commits

File: scripts/release_sdk.sh (hunk 3 above)
Comment (s-urbaniak) [reply scripts/release_sdk.sh]: `set -euxo pipefail`

File: scripts/md5sum.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+const ALGORITHM = 'md5';
+const ARG_POS   = 2;
+const FILENAME  = process.argv[ARG_POS];
+
+const algoSum     = createHash(ALGORITHM);
+const inputStream = FILENAME ? ReadStream(FILENAME) : process.stdin;
+
+inputStream.on('data', (data) => algoSum.update(data));
+inputStream.on('end', () => console.log(algoSum.digest('hex')));
```
[the remark this answers is not in the document]
Comment (s-urbaniak) [reply scripts/md5sum.sh]: also I suggest we use `sha256` instead.

File: scripts/release_sdk.sh (hunk 1 above)
Comment (ibihim) [reply scripts/release_sdk.sh]: it has, applied

**Pull request merged**